examples/prometheus/prometheus.yaml

```diff
@@ -0,0 +1,201 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: prometheus
+  annotations:
+    "openshift.io/display-name": Prometheus
+    description: |
+      A monitoring solution for an OpenShift cluster - collect and gather metrics from nodes, services, and the infrastructure.
+      You will need to grant access to the prometheus service account to access nodes, metrics endpoints, and services across
+      namespaces - use `oadm policy add-cluster-role-to-user system:serviceaccount:NAMESPACE:prometheus cluster-reader`.
```

Comment:
should be `oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:NAMESPACE:prometheus`